specification/cognitiveservices/data-plane/Face/v1.0/Face.json

```diff
@@ -1568,50 +1207,110 @@
         ]
       }
-    },
-    "/persongroups/{personGroupId}/persons/{personId}/persistedFaces?overload=stream": {
+    }
+  },
+  "x-ms-paths": {
+    "/verify?overload=persongroup": {
       "post": {
-        "description": "Add a representative face to a person for identification. The input face is specified as an image with a targetFace rectangle.",
-        "operationId": "Person_AddFaceFromStream",
+        "description": "Verify whether two faces belong to a same person. Compares a face Id with a Person Id",
+        "operationId": "Face_VerifyWithPersonGroup",
         "parameters": [
           {
-            "name": "personGroupId",
-            "in": "path",
-            "description": "Specifying the person group containing the target person.",
+            "name": "body",
+            "description": "Request body for verifying two faces in a person group",
+            "in": "body",
             "required": true,
-            "type": "string"
+            "x-ms-client-flatten": true,
+            "schema": {
+              "$ref": "#/definitions/VerifyPersonGroupRequest"
+            }
+          }
+        ],
+        "consumes": [
+          "application/json"
+        ],
+        "responses": {
+          "200": {
+            "description": "A successful call returns the verification result.",
+            "schema": {
+              "$ref": "#/definitions/VerifyResult"
+            }
           },
+          "default": {
+            "description": "Error response.",
+            "schema": {
+              "$ref": "#/definitions/APIError"
+            }
+          }
+        },
+        "produces": [
+          "application/json"
+        ]
+      }
+    },
+    "/detect?overload=stream": {
+      "post": {
+        "description": "Detect human faces in an image and returns face locations, and optionally with faceIds, landmarks, and attributes.",
+        "operationId": "Face_DetectInStream",
+        "parameters": [
           {
-            "name": "personId",
-            "in": "path",
-            "description": "Target person that the face is added to.",
-            "required": true,
-            "type": "string"
+            "name": "returnFaceId",
+            "in": "query",
+            "description": "A value indicating whether the operation should return faceIds of detected faces.",
+            "type": "boolean",
+            "default": true
           },
           {
-            "name": "userData",
+            "name": "returnFaceLandmarks",
             "in": "query",
-            "description": "User-specified data about the target face to add for any purpose. The maximum length is 1KB. ",
-            "type": "string"
+            "description": "A value indicating whether the operation should return landmarks of the detected faces.",
+            "type": "boolean",
+            "default": false
           },
           {
-            "name": "targetFace",
+            "name": "returnFaceAttributes",
```

Comment:
>returnFaceAttributes [](start = 21, length = 20)

As this parameter is used in other operation too, does it make sense to define it outside the operation, and make the operations reference the definition?

Reply:
Fixed